### PR TITLE
Update graph-importer worker with source_repo in arguments.

### PR DESCRIPTION
**Pull request by GeetikaBatra**
Signed-off-by: Geetika Batra gbatra@redhat.com

### CODE REVIEW

File: f8a_worker/workers/graph_importer.py [hunk 1]
```diff
@@ -35,7 +35,8 @@ def execute(self, arguments):
             {
                         'ecosystem': arguments['ecosystem'],
```
Comment (msrb) [inline f8a_worker/workers/graph_importer.py]: `arguments['ecosystem']` could be for example `redhat-maven`. You need to do something like https://github.com/msrb/fabric8-analytics-worker/blob/8f069e308df89ebdab8c89f0e63adea1cd52294d/f8a_worker/workers/CVEchecker.py#L218-L219 to translate ecosystem string to Ecosystem object and from there you should be able to find corresponding ecosystem backend ('maven' for ecosystem 'redhat-maven').

File: f8a_worker/workers/graph_importer.py [hunk 2]
```diff
@@ -35,7 +35,8 @@ def execute(self, arguments):
...
                         'name': arguments['name'],
-                        'version': arguments.get('version')
+                        'version': arguments.get('version'),
+                        'source_repo': arguments.get('source_repo', None)
```
Comment (msrb) [inline f8a_worker/workers/graph_importer.py]: There will be no `source_repo` arg :) `arguments.get('ecosystem')` is what graph considers `source_repo`.

File: f8a_worker/workers/graph_importer.py (hunk 1 above)
Comment (GeetikaBatra) [reply f8a_worker/workers/graph_importer.py]: @msrb, How would we determine what is the backend then?

File: f8a_worker/workers/graph_importer.py [hunk 3]
```diff
@@ -31,11 +33,14 @@ def execute(self, arguments):
         self._strict_assert(arguments.get('name'))
         self._strict_assert(arguments.get('document_id'))
 
+        rdb = StoragePool.get_connected_storage('BayesianPostgres')
+        ecosystem = Ecosystem.by_name(rdb.session, arguments.get('ecosystem')).name
```
Comment (msrb) [inline f8a_worker/workers/graph_importer.py]: If `arguments['ecosystem']` is `redhat-maven`, what will be the value stored in `ecosystem` variable? :)

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
LGTM, thanks @GeetikaBatra :wink: 

Just waiting for CI to be green,

**Pull request merged**